dev/integration_tests/wide_gamut_test/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think you need to add all these files.  `flutter create --no-overwrite` is used.

Reply:
Removed